src/parser/transform/transform_ddl.cpp

```diff
@@ -26,13 +26,36 @@ std::unique_ptr<Statement> Transformer::transformAlterTable(
     }
 }
 
-std::unique_ptr<Statement> Transformer::transformCreateNodeTable(
-    CypherParser::KU_CreateNodeTableContext& ctx) {
-    auto tableName = transformSchemaName(*ctx.oC_SchemaName());
+std::string Transformer::getPKName(CypherParser::KU_CreateNodeTableContext& ctx) {
+    int pkCount = 0;
     std::string pkName;
+    auto& propertyDefinitions = *ctx.kU_PropertyDefinitions();
+    for (auto& definition : propertyDefinitions.kU_PropertyDefinition()) {
+        if (definition->PRIMARY() && definition->KEY()) {
+            pkCount++;
+            pkName = transformPrimaryKey(*definition->kU_ColumnDefinition());
+        }
+    }
     if (ctx.kU_CreateNodeConstraint()) {
+        pkCount++;
```

Comment:
We probably doesn't want to error in the following case

```
CREATE NODE TABLE (id INT64 PRIMARY KEY,  PRIMARY KEY(id));
```

Try handle this by also checking if primary key names are the name.